fastlane/jetpack_metadata/android/sv-SE/short_description.txt

```diff
@@ -0,0 +1 @@
+Bygg en WordPress-webbplats, skapa inlägg, lägg till foton och se statistik!
```

Comment:
Confirmed to have been taken into account by PlayStore:
<img width="1582" alt="image" src="https://user-images.githubusercontent.com/216089/188201418-e63e9488-bcec-48f8-a9a2-1adb1c8ef96c.png">